gradle/documentation/render-javadoc.gradle

```diff
@@ -370,7 +370,7 @@ class RenderJavadocTask extends DefaultTask {
       opts << [ '-linkoffline', url, dir ]
     }
 
-    opts << [ '--release', 11 ]
+    opts << [ '--release', project.minJavaVersion.toString() ]
```

Comment:
https://github.com/apache/solr/blob/2eb9b21d434da748d26ec82cc2b76aa1fd41237b/gradle/globals.gradle#L175 approximately looks to be where `project.minJavaVersion` comes from.

Reply:
That's correct, each project wil have this variable set, so javadoc for SolrJ will use 11 while core may use 17 eventually.